labcontrol/gui/js_tests/addPlateModal_tests.js

```diff
@@ -0,0 +1,154 @@
+// Requires
```

Comment:
Is there a documentation standard for js and HTML files akin to how we document Python methods? We should probably adopt one, although I think that can be a separate issue.

Reply:
Entirely agree re the need for an overall javascript documentation standard.  Open to suggestions ... maybe we should look at what Yoshiki does in EMPeror.  I just added issue #529 to capture.